examples/lambda-functions/get-all-items.ts

```diff
@@ -0,0 +1,87 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult, Context } from 'aws-lambda';
+import { Metrics } from '@aws-lambda-powertools/metrics';
+import { Logger } from '@aws-lambda-powertools/logger';
+import { Tracer } from '@aws-lambda-powertools/tracer';
+import { DynamoDB } from 'aws-sdk';
```

Comment:
Let's import only this client instead: `import { DocumentClient } from 'aws-sdk/clients/dynamodb';`

Reply:
changed it for all lambdaHandlers